DemoApp/BannerView.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChatUI
+import UIKit
+
+final class BannerView: UIView {
+    private let label = UILabel()
+    
+    override func didMoveToSuperview() {
+        super.didMoveToSuperview()
+        setUpLayout()
+        defaultAppearance()
+    }
```

Comment:
This a pattern from the SDK and I don't think we should use it here. The SDK for example takes care of the fact that this is called just once, which here doesn't happen. I think it's better to call these methods from a method which is guaranteed to be called just once.

Reply:
Now it's called once from init